src/contexts/Guilds/filters.tsx

```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext } from 'react';
+import { useMenu } from '../../hooks/Guilds/useMenu';
+
+const FilterContext = createContext(null);

```

Comment:
I had similiar thoughts on doing this for dxvote as well. Maybe we do the same in the future to keep consistency?